docs/syntax.rst

```diff
@@ -59,6 +64,29 @@ meaningful, FTL allows for special quote delimited strings as can be seen in
 the ``more-info`` entity.
 
 
+Entity References
+=================
+
+::
+
+    brandName = Loki
+    installing = Installing { brandName }.
+
+    menu-save = Save
+    help-menu-save = Click "{ menu-save }" to save the file.
+
+In FTL strings may use special syntax to incorporate small pieces of
```

Comment:
Let's use this opportunity to fix the wording here:

    Strings in FTL may use...